Homework-6/Controllers/StudentController.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Homework_6.Data;
+using Homework_6.Dto;
+using Homework_6.Model;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Homework_6.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class StudentsController : ControllerBase
+    {
+        [HttpPost]
+        public ActionResult<int> CreateStudent([FromBody] StudentRequestDto studentRequest)
+        {
+            var student = new Student
+            {
+                Name = studentRequest.Name,
+                Age = studentRequest.Age,
+                Grade = studentRequest.Grade
+            };
+
+            student.Id = StudentData.GetNextId();
+            StudentData.Students.Add(student);
```

Comment:
all this code should have been inside a service class. Controller layer should only be able to talk to service class